docs/hydra/configure-deploy.mdx

```diff
@@ -312,11 +320,12 @@ docker run --rm -it \
     --client-secret some-secret \
     --scope openid,offline,photos.read
 
-Setting up callback listener on http://localhost:9010/callback
+Setting up home route on http://127.0.0.1:9010/
+Setting up callback listener on http://127.0.0.1:9010/callback
 Press ctrl + c on Linux / Windows or cmd + c on OSX to end the process.
-If your browser doesn't open automatically, navigate to:
+If your browser does not open automatically, navigate to:
```

Comment:
```suggestion
If your browser doesn't open automatically, navigate to:
```

Probably need to fix this in the CLI / API docs at some point. I think the contraction is much more natural sounding 🤷